tests/ui/PureReportActionItemTest.tsx

```diff
@@ -163,7 +163,7 @@ describe('PureReportActionItem', () => {
             await waitForBatchedUpdatesWithAct();
 
             expect(screen.getByText(actorEmail)).toBeOnTheScreen();
-            expect(screen.getByText(translateLocal('iou.submitted'))).toBeOnTheScreen();
+            expect(screen.getByText(translateLocal('iou.submitted', {}))).toBeOnTheScreen();
```

Comment:
@gijoe0295, should we add an additional test case with a memo (message) included?